week-2/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >=80) {
+    return "A";
+  } else if (mark < 80 && mark >60 ){
```

Comment:
You don't need to check `mark < 80` here since we already know that `mark >= 80` is `false` from the first branch of the `if` statement.

Reply:
Oh ,yes I see now